assets/src/dashboard/components/storyAnimation/test/provider.js

```diff
@@ -102,25 +102,48 @@ describe('StoryAnimation.Provider', () => {
       } = result.current;
 
       getAnimationParts(target).forEach((animationPart, i) => {
-        expect(animationPart).toStrictEqual({ type: type[i], args });
+        expect(animationPart).toStrictEqual({
+          type: type[i],
+          args: { ...args, element: undefined },
```

Comment:
If we don't pass in `elements` then each animation part will simply get `undefined` passed in for its element.